---
format:
  revealjs:
    slide-number: c/t
    width: 1600
    height: 900
    css: ["theme/theme.css"]
    theme: [simple,theme/mystyle.scss]
    echo: true
editor: source
freeze: true
pdf-separate-fragments: true
---
# {.title-slide visibility=hidden}

<!-- begin ----------------------------- -->
# Leveraging deep learning models to enable prediction of omics data in low power settings {background-color="white"}
<br>

<h2> ICSA 2024, Nashville</h2>

<h3>2024-06-18 Wednesday</h3> 

<h3>Hae Kyung Im</h3>   
<!-- end ----------------------------- -->

<!-- begin ----------------------------- -->
## Why do we want to predict omics traits from genotype? {.nostretch 
background-image=assets/slide-background.svg}

. . . 

<br>
To explain mechanisms of GWAS loci

. . . 

GWAS discoveries continue to grow but causal genes are not known

![](assets/gwas-catalog-2024-04-10.png){width=50%}

. . .  

288,566 SNP/trait pairs from 5,604 studies as of 4/2024.
<!-- <small>downloaded April 2024 from GWAS Catalog </small> -->

<!-- end ----------------------------- -->


<!-- begin ----------------------------- -->
## TWAS uses GWAS data to find genes associated with trait {background-image=assets/slide-background.svg}

![](assets/gwas-predixcan-diagram){width=1200}

::: {.absolute bottom=0 left=0 .fragment .fade-out}
![](assets/blank-rectangle.png){width="1600" height="380" fig-alt="blank"}
:::
<!-- end ----------------------------- -->


<!-- begin ----------------------------- -->
## Gene expression can be predicted from genotype {background-image=assets/slide-background.svg}

![](assets/reference-transcriptome-gtex.png)

<!-- begin ----------------------------- -->
## Some genes are very well predicted {background-image=assets/slide-background.svg}

![](assets/performance-top-genes.png)

<!-- begin ----------------------------- -->
## Accuracy of prediction depends on heritability {.nostretch  background-image=assets/slide-background.svg}

![](assets/performance-all-genes.png){width=75%}


<!-- begin ----------------------------- -->
## Advantages of gene level associations {background-image=assets/slide-background.svg }

- Reduced multiple testing burden (from 10e6 to 10e4)
- The function of genes are much better annotated than SNPs
- Validation in other model systems is possible
- Reverse causality issues is less problematic because disease status doesn’t affect germline genetic markers 
- It provides the direction of effects, i.e. whether up or down regulation of a gene - increases the risk of a disease
- The candidate causal gene is a good target for drug development

<!-- begin ----------------------------- -->
## PrediXcan results identify drug repositioning candidates {
.nostretch 
background-image=assets/slide-background.svg 
}

![](assets/so-drug-repurposing.png){width=75%}

:::aside
So et al, Nature Neuroscience 2017
::: 
<!-- end ----------------------------- -->

<!-- begin ----------------------------- -->
## PrediXcan Results Are Predictive of Drug Target Success {
background-image=assets/slide-background.svg}

![](assets/successful-drugs-higher-twas-signal.png){width=65%}
<!-- end ----------------------------- -->

<!-- begin ----------------------------- -->
## Limitations of TWAS  {
background-image=assets/slide-background.svg 
visibility=hidden
}

:::{.columns}

:::{.column width=75%}

- Not very good at predicting low heritability genes which are more likely to be disease relevant
- Predictions don’t transfer well across ancestries
- Effect of rare variation is missed
- Steady state eQTL may not be disease relevant

:::
:::
<!-- end ----------------------------- -->


<!-- begin ----------------------------- -->
## Limitations of TWAS {background-image=assets/slide-background.svg .nostretch }

<br><br>

::::{.columns}

:::{.column width="70%"}

- bulk RNA may not reveal disease relevant regulation
<br>
<br>

- context-specific regulation is likely to be more relevant
<br>
<br>

- transcription factor binding alteration may affect disease

:::

:::{.column width="30%"}

:::{.fragment}
- scPrediXcan
<br>
<br>

- context-Xcan
<br>
<br>

- TFXcan
:::

:::

::::

:::{.fragment}
<br><br>
[Generating data with sufficient reads and sample sizes is too costly]{style="font-size: 1.3em;"}
:::
<!-- end ----------------------------- -->

<!-- begin ----------------------------- -->
# Can deep learning help? {background-color="white"}
<!-- end ----------------------------- -->

<!-- begin ----------------------------- -->
## Enformer predicts epigenetic features from DNA sequence {background-image=assets/slide-background.svg .nostretch }
![](assets/enformer-input-output.png){width=75%}

[250 million parameters, 4.6K TPU hours]{style="font-size: 0.7em"}

:::{.absolute right=0 bottom=0}
[$~~~~~~~~~~~~~~~~$ Avsec et al Nature Methods 2021]{style="font-size: 0.5em"}
:::
<!-- end ----------------------------- -->

<!-- begin ----------------------------- -->
## Enformer predicts epigenetic features from DNA sequence {background-image=assets/slide-background.svg .nostretch }
$~~~~~~$ ![](assets/enformer-track-performance.png){width=85%}

:::{.absolute right=0 bottom=0}
[$~~~~~~~~~~~~~~~~$ Avsec et al Nature Methods 2021]{style="font-size: 0.5em"}
:::
<!-- end ----------------------------- -->

<!-- begin ----------------------------- -->
## Enformer trained to perform well across the genome {background-image=assets/slide-background.svg .nostretch }
$~~~~~~~~~~~~~~~~$ ![](assets/enformer-dimensions-genomic-population-01.png){width=99%}
<!-- end ----------------------------- -->

<!-- begin ----------------------------- -->
## Enformer trained to perform well across the genome {background-image=assets/slide-background.svg .nostretch }
$~~~~~~~~~~~~~~~~$ ![](assets/enformer-dimensions-genomic-population-02.png){width=99%}
<!-- end ----------------------------- -->

<!-- begin ----------------------------- -->
## Enformer trained to perform well across the genome {background-image=assets/slide-background.svg .nostretch }
$~~~~~~~~~~~~~~~~$ ![](assets/enformer-dimensions-genomic-population-03.png){width=99%}
<!-- end ----------------------------- -->

<!-- begin ----------------------------- -->
# Can enformer predict variation across individuals?  {background-color="white"}
<!-- end ----------------------------- -->


<!-- begin ----------------------------- -->
## Enformer can predict variation across individuals - PEX6 {background-image=assets/slide-background.svg .nostretch }
![](assets/pex6-enformer.png){width=70%}

::: {.absolute bottom=-90 right=-18}
![](assets/temi-sai.png){width=400}
:::

<!-- end ----------------------------- -->

<!-- begin ----------------------------- -->
## but correlation can be negative - NUDT2 {background-image=assets/slide-background.svg .nostretch }
![](assets/nudt2-enformer.png){width=70%}

::: {.absolute bottom=-90 right=-18}
![](assets/temi-sai.png){width=400}
:::
<!-- end ----------------------------- -->



<!-- begin ----------------------------- -->
## Personal transcriptome poorly explained {background-image=assets/slide-background.svg .nostretch }

![](assets/enformer-problem-NG.png)

<!-- end ----------------------------- -->


<!-- begin ----------------------------- -->
## Mean correlation across genes vs. across individuals {background-image=assets/slide-background.svg .nostretch }

![](assets/cross-individual-enformer-comparison-huang.png){width=60%}

<!-- end ----------------------------- -->


<!-- begin ----------------------------- -->
## Average correlation hides utility of unsigned of prediction {background-image=assets/slide-background.svg .nostretch }

![](assets/enformer-vs-predixcan.png){width=50%}

:::{.absolute right=0 bottom=0}
[Huang et al 2023 NG]{style="font-size: 1em"}
:::
<!-- end ----------------------------- -->

<!-- begin ----------------------------- -->
## Average correlation hides utility of unsigned of prediction {background-image=assets/slide-background.svg .nostretch }

![](assets/glass_people.png){width=95%}

:::{.absolute right=0 bottom=0}
[https://what-if.xkcd.com/6/]{style="font-size: 1em"}
:::
<!-- end ----------------------------- -->





<!-- begin ----------------------------- -->

# scPrediXcan {background-color="white"}

uses single cell data to identify cell-type specific causal genes

<!-- end ----------------------------- -->



<!-- begin ----------------------------- -->
## Overview of scPrediXcan Framework {background-image=assets/slide-background.svg .nostretch }

![](assets/scpredixcan-overview.png){width=80%}

<!-- ::: {.absolute bottom=-70 right=-70} -->
::: {.absolute bottom=0 right=0}
![](assets/charles-mengjie.png){width=300}
:::
<!-- end ----------------------------- -->



<!-- begin ----------------------------- -->
## scPrediXcan predicts cell type specific expression {background-image=assets/slide-background.svg .nostretch }

![](assets/scpred-performance-top-genes.png){width=65%}

::: {.absolute bottom=-70 right=-70}
![](assets/charles-mengjie.png){width=400}
:::
<!-- end ----------------------------- -->







<!-- begin ----------------------------- -->
## scPrediXcan outperforms traditional PrediXcan {background-image=assets/slide-background.svg .nostretch }

![](assets/scpredixcan-t2d.png){width=100%}

::: {.absolute bottom=-70 right=-70}
![](assets/charles-mengjie.png){width=400}
:::
<!-- end ----------------------------- -->



<!-- begin ----------------------------- -->
## scPrediXcan identifies more likely type 2 diabetes genes {background-image=assets/slide-background.svg .nostretch visibility=hidden}

![](assets/scPrediXcan-vs-bulk-PrediXcan-causal.png){width=100%}

::: {.absolute bottom=-70 right=-70}
![](assets/charles-mengjie.png){width=400}
:::
<!-- end ----------------------------- -->

<!-- begin ----------------------------- -->
## scPrediXcan outperforms bulk PrediXcan {background-image=assets/slide-background.svg .nostretch  visibility=hidden}

![](assets/scPrediXcan-vs-bulk-PrediXcan-qqplot.png){width=60%}

::: {.absolute bottom=-70 right=-70}
![](assets/charles-mengjie.png){width=400}
:::
<!-- end ----------------------------- -->






<!-- begin ----------------------------- -->
## Next steps {background-image=assets/slide-background.svg .nostretch visibility=hidden}
:::{.incremental}

- cross species integration (Sabrina Mi/UCSD)

- DNA language model training (Henry Raeder rotation project)

- Multi-omic language model training (Rodrigo Bonazzola U Leeds)

- Methylation prediction (Evan Wu Bioinformatics Core et al)

- Summary based mixed effects model (Sabrina Mi)
:::
<!-- end ----------------------------- -->


<!-- begin ----------------------------- -->
## Conclusions {background-image=assets/slide-background.svg .nostretch }
:::{.incremental}

- TWAS help us identify causal genes but has limitations
- Cell type specific predictors can enhance TWAS
  - but we have small sample sizes and sparse data
- Deep learning methods allow us train models with limited sample sizes
- scPrediXcan combines population based and deep learning overcoming the issues of traditional TWAS




:::
<!-- end ----------------------------- -->



<!-- begin ----------------------------- -->
## Thank you {background-image=assets/slide-background.svg .nostretch }
![](assets/acknowledgement.png)
<!-- end ----------------------------- -->







<!-- begin ----------------------------- -->

# TFXcan {background-color="white"}

identifies TFs disrupted by GWAS loci using genetic predictors of TF binding

<!-- end ----------------------------- -->


<!-- begin ----------------------------- -->
## Enpact Method - Training {background-image=assets/slide-background.svg .nostretch }

![](assets/enpact-method.png)

For training, population average is used, 
so only a handful of samples are needed

<!-- end ----------------------------- -->



<!-- begin ----------------------------- -->
## Validation of Enpact prediction compared to observed {background-image=assets/slide-background.svg .nostretch }

![](assets/TF-enpact-vs-observed.png){width=99%}



::: {.absolute bottom=-90 right=-18}
![](assets/temi.png){width=200}
:::

. . . 

[Prediction performance is comparable to population-based method with 130 samples]{style="font-size=2em;"}

<!-- end ----------------------------- -->

<!-- begin ----------------------------- -->
## Validation of Enpact prediction compared to other predictors  {background-image=assets/slide-background.svg .nostretch }

![](assets/enpact-vs-cwas.png)

::: {.absolute top=100 right=10}
Top site by correlation
:::


::: {.absolute bottom=-90 right=-18}
![](assets/temi.png){width=200}
:::
<!-- end ----------------------------- -->


<!-- begin ----------------------------- -->
## Most enpact predictors correlate significantly with CWAS {background-image=assets/slide-background.svg .nostretch }

![](assets/TF-enpact-vs-cwas-01.png)

::: {.absolute bottom=-90 right=-18}
![](assets/temi.png){width=200}
:::
<!-- end ----------------------------- -->

<!-- begin ----------------------------- -->
## Enpact CWAS correlation sign is also inconsistent {background-image=assets/slide-background.svg .nostretch }  

![](assets/TF-enpact-vs-cwas-02.png)

::: {.absolute bottom=-90 right=-18}
![](assets/temi.png){width=200}
:::
<!-- end ----------------------------- -->

<!-- begin ----------------------------- -->
## TFXcan identifies TFBinding associated with disease  {background-image=assets/slide-background.svg .nostretch visibility=hidden}
![](assets/tfxcan.svg){width=90%}

::: {.absolute bottom=-90 right=-18}
![](assets/temi.png){width=200}
:::

<!-- end ----------------------------- -->


<!-- begin ----------------------------- -->
## AR binding sites associated with prostate cancer risk  {background-image=assets/slide-background.svg .nostretch }
![](assets/tfxcan-manhattan-2.svg){width=120%}

::: {.absolute bottom=-90 right=-18}
![](assets/temi.png){width=200}
:::

<!-- end ----------------------------- -->


<!-- begin ----------------------------- -->
## Concordance with CWAS results  {background-image=assets/slide-background.svg .nostretch  visibility=hidden}
![](assets/ar-tfxcan-cwas-venn.svg){width=50%}


::: {.absolute bottom=-90 right=-18}
![](assets/temi.png){width=200}
:::
<!-- end ----------------------------- -->


<!-- begin ----------------------------- -->
## Preliminary Tenerife plot in Breast Cancer loci  {background-image=assets/slide-background.svg .nostretch }


![](assets/ovarian-tfxcan.png){width=120%}



::: {.absolute bottom=-70 right=-18}
![](assets/sofia.png){width=200}
:::

:::{.absolute bottom=-60 right=200}
 [Sofia Salazar]{style="font-size: 0.6em"}
:::
<!-- end ----------------------------- -->



<!-- begin ----------------------------- -->
## Context specific expression prediction {background-image=assets/slide-background.svg .nostretch visibility=hidden}

![](assets/aracena-data.png)
[$~~~~~~~~~~~~~~~~$ Aracena et al 2023 NG]{style="font-size: 0.5em"}


::: {.absolute bottom=-70 right=-18}
![](assets/sai.png){width=200}
:::

:::{.absolute bottom=-60 right=200}
 [Sai Gona]{style="font-size: 0.6em"}
:::
<!-- end ----------------------------- -->

<!-- begin ----------------------------- -->
<!-- ## Transcription factor binding prediction (Temidayo Adeluwa){background-image=assets/slide-background.svg .nostretch } -->

<!-- ::: {.absolute bottom=-70 right=-18} -->
<!-- ![](assets/temi.png){width=200} -->
<!-- ::: -->
<!-- end ----------------------------- -->



<!--  ----------------------------- -->
<!-- acknowledgements ----------------------------- -->
<!-- :::{.absolute bottom=-60 right=200} -->
<!--  [Charles Zhou]{style="font-size: 0.6em"} -->
<!-- ::: -->
<!-- end ----------------------------- -->


